misc-utils/lsfd-unkn.c

```diff
@@ -1382,6 +1385,27 @@ static const struct anon_ops *anon_probe(const char *str)
 	return &anon_generic_ops;
 }
 
+static bool has_pidfs = false;
+static dev_t pidfs_dev;
+
+static void unkn_init_class(void)
+{
+#ifdef __NR_pidfd_open
+	int fd = syscall(__NR_pidfd_open, getpid(), 0);
```

Comment:
`include/pidfd-utils.h` already has an implementation of `pidfd_open()`.
Using it will allow a `lsfd` built against 6.8 headers work correctly against 6.9 at runtime.

Reply:
Done.  It should be {the kernel version when pidfd was added} (I cannot remember the x.y numbers) instead of 6.8 though because the API remains same from 6.8 to 6.9, just the internal representation of pidfds are changed.